arcgis-ios-sdk-samples/Display information/Sketch on the map/Sketch.storyboard

```diff
@@ -20,38 +21,34 @@
                             <toolbar clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="45u-o5-Qgv">
                                 <rect key="frame" x="0.0" y="813" width="414" height="49"/>
                                 <items>
-                                    <barButtonItem id="Mme-ds-Pgy">
-                                        <segmentedControl key="customView" opaque="NO" tag="55" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="top" segmentControlStyle="bar" selectedSegmentIndex="1" id="X7k-S7-bpe">
-                                            <rect key="frame" x="20" y="11" width="204" height="32"/>
-                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
-                                            <segments>
-                                                <segment title="" image="Point"/>
-                                                <segment title="" image="Polyline"/>
-                                                <segment title="" image="FreehandPolyline"/>
-                                                <segment title="" image="Polygon"/>
-                                                <segment title="" image="FreehandPolygon"/>
-                                            </segments>
-                                            <connections>
-                                                <action selector="geometryValueChanged:" destination="DMe-yp-yH3" eventType="valueChanged" id="UPW-qg-uR6"/>
-                                            </connections>
-                                        </segmentedControl>
-                                    </barButtonItem>
-                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="heC-81-cSK"/>
-                                    <barButtonItem image="Undo" id="Xfw-dX-zVQ">
+                                    <barButtonItem systemItem="undo" id="Xfw-dX-zVQ">
                                         <connections>
                                             <action selector="undo" destination="DMe-yp-yH3" id="hpK-dm-9G0"/>
                                         </connections>
                                     </barButtonItem>
-                                    <barButtonItem image="Redo" id="fQO-v5-LKM">
+                                    <barButtonItem systemItem="redo" id="fQO-v5-LKM">
                                         <connections>
                                             <action selector="redo" destination="DMe-yp-yH3" id="c6W-St-RTy"/>
                                         </connections>
                                     </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="32P-6u-mlV"/>
+                                    <barButtonItem systemItem="add" id="h9e-Af-91p">
```

Comment:
Add seems like an odd choice here as selecting it doesn't actually add anything. Since it allows the user to change the mode, a better label for this button might be Mode.